frb_codegen/src/commands.rs

```diff
@@ -121,19 +121,20 @@ fn cbindgen(
             "stdint.h".to_string(),
             "stdlib.h".to_string(),
         ],
+        defines: [("target_family = 'wasm'".into(), "DEFINE_WASM".into())].into(),
```

Comment:
Not very sure about this line - Will it do harm when targeting non-wasm?

Reply:
This is mostly a no-op, I meant for it to silence the warnings from cbindgen but it isn't really effective.

Reply:
I see. If it is not working then maybe we should comment it out, o/w future readers of this code may get confused